### ISSUE TITLE
Correct description of required model-output subdirectory structure

### ISSUE THREAD

**Issue by annakrystalli** [Infectious-Disease-Modeling-Hubs/hubDocs#116]
Currently the documentation on the expected model-output subdirectory structure on page `/user-guide/model-output.html` is incorrect

It show the required structure to erroneously be:

```
team1-modela
   <round-id1>.csv (or parquet, etc)
   <round-id2>.csv (or parquet, etc)
team1-modelb
   <round-id1>.csv (or parquet, etc)
team2-modela
   <round-id1>.csv (or parquet, etc)
```

when it should be:
```
team1-modela
   <round-id1>-<model_id>.csv (or parquet, etc)
   <round-id2>-<model_id>.csv (or parquet, etc)
team1-modelb
   <round-id1>-<model_id>.csv (or parquet, etc)
team2-modela
   <round-id1>-<model_id>.csv (or parquet, etc)
```

**Comment by annakrystalli** [comment]
This might also be a good place to re-iterate that `model_id` = `team_abbr-model_abbr`.

**Comment by mmkerr** [comment]
Resolved with PR #119
